sdk/python/feast/feature_view.py

```diff
@@ -60,16 +69,23 @@ def __init__(
         name: str,
         entities: List[str],
         ttl: Optional[Union[Duration, timedelta]],
-        input: Union[BigQuerySource, FileSource],
+        input: Optional[Union[BigQuerySource, FileSource]],
+        batch_source: Optional[Union[BigQuerySource, FileSource]] = None,
```

Comment:
What's the difference between input and batch_source?

Reply:
I want to deprecate `input` - `batch_source` is supposed to supercede `input`. The reason I'm adding a new field is that because we need to differentialte stream and batch sources. This also matches the API for FeatureTables.